city/models.py

```diff
@@ -0,0 +1,29 @@
+from sqlalchemy import Column, DateTime, Float, ForeignKey, Integer, String
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class DBCity(Base):
+    __tablename__ = "city"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(64), nullable=False)
+    additional_info = Column(String(255), nullable=False)
```

Comment:
The `additional_info` column is set as `nullable=False`. Consider whether this field should be optional, as not all cities might have additional information.